README.md

```diff
@@ -31,6 +34,35 @@ Document<InvoiceV4Inference> invoiceDocument = mindeeClient.parse(
   documentToParse);
 ```
 
+### Region-Specific Documents
+```java
+import com.mindee.parsing;
+import com.mindee.parsing.us.bankcheck;
+
+MindeeClient mindeeClient = new MindeeClient(new MindeeSetings("<MINDEE API KEY>"));
+
+DocumentToParse documentToParse = mindeeClient.loadDocument(new File("src/main/resources/invoices/invoice1.pdf"));
+
+Document<BankCheckV1Inference> bankCheckDocument = mindeeClient.parse(
+  BankCheckV1Inference.class,
+  documentToParse);
+```
+
+### Custom Documents (API Builder)
+```java
+import com.mindee.parsing;
+import com.mindee.parsing.custom;
+
+MindeeClient mindeeClient = new MindeeClient(new MindeeSetings("<MINDEE API KEY>"));
+CustomEndpoint customEndpoint = new CustomEndpoint("myProductName", "myAccountName", "myModelVersion");
```

Comment:
```suggestion
CustomEndpoint customEndpoint = new CustomEndpoint("myProductName", "myAccountName");
```
Adding the version is more of an advanced usage, in the TL;DR we just want to make sure it will work.